hw/mcu/nordic/nrf5340/include/mcu/cmsis_nvic.h

```diff
@@ -0,0 +1,28 @@
+/* mbed Microcontroller Library - cmsis_nvic
+ * Copyright (c) 2009-2011 ARM Limited. All rights reserved.
+ *
+ * CMSIS-style functionality to support dynamic vectors
+ */
+
+#ifndef _CMSIS_NVIC_H_
+#define _CMSIS_NVIC_H_
+
+#include <stdint.h>
+#include <nrf.h>
+
+#define NVIC_NUM_VECTORS      (16 + 39)   /* CORE + MCU Peripherals */
```

Comment:
Are you sure 39 is correct number?

Reply:
no idea where I took this from :-) will fix it